x-pack/plugins/fleet/common/constants/routes.ts

```diff
@@ -123,6 +123,7 @@ export const AGENT_API_ROUTES = {
   UPGRADE_PATTERN: `${API_ROOT}/agents/{agentId}/upgrade`,
   BULK_UPGRADE_PATTERN: `${API_ROOT}/agents/bulk_upgrade`,
   CURRENT_UPGRADES_PATTERN: `${API_ROOT}/agents/current_upgrades`,
+  LIST_TAGS_PATTERN: `${API_ROOT}/agents/tags`,
```

Comment:
Could you add this new endpoint to the openapi specs?

Reply:
sure